### PR TITLE
KEYCLOAK-18092 Upgrade to 13.0.0 failing because of status.externalURL field

### PR DESCRIPTION
**Pull request by vmuzikar**
## JIRA ID
https://issues.redhat.com/browse/KEYCLOAK-18092

## Checklist:
- [ ] Automated Tests
- [ ] [Documentation](https://github.com/keycloak/keycloak-documentation) changes if necessary
- [ ] (If required) Discussed on [Keycloak DEV](https://groups.google.com/forum/#!forum/keycloak-dev)

### CODE REVIEW

File: deploy/crds/keycloak.org_keycloakrealms_crd.yaml [hunk 1]
```diff
@@ -1141,6 +1138,9 @@ spec:
                         type: string
                     type: object
                   type: array
+                userManagedAccessAllowed:
+                  description: User Managed Access Allowed
+                  type: boolean
```
Comment (vmuzikar) [inline deploy/crds/keycloak.org_keycloakrealms_crd.yaml:1143]: This is an implicit fix (made by `make code/gen`). In #346 the  change to the CRD was apparently made manually and at a wrong place (in clients section, not realm).

Comment (slaskawi) [reply deploy/crds/keycloak.org_keycloakrealms_crd.yaml:1143]: It's fine.

**Review by slaskawi — APPROVED** [review]

**Comment by slaskawi** [comment]
This commit should be cherry-picked to the REL branch as well.

**Comment by slaskawi** [comment]
@stianst Please advice how you include this in 13.0.1 in your release machinery.

**Pull request merged**